futures/tests/basic.rs

```diff
@@ -14,7 +14,7 @@ async fn sleep(times: Vec<Duration>) -> i32 {
         task::yield_now().await;
     }
     // Just random result
-    10
+    10_i32
```

Comment:
do we have to specify the type on literals from now on? clippy lints against this?

Reply:
That's correct we have this clippy lint enabled generally.

Reply:
We don't actually, and we didn't. But I will enable it.

Reply:
Interesting, maybe it was disabled recently. I haven't noticed.